docker/workers/Dockerfile

```diff
@@ -46,4 +46,4 @@ ENV GOVUK_APP_DOMAIN ''
 ENV GOVUK_WEBSITE_ROOT ''
 
 ARG RAILS_ENV=production
-CMD bin/delayed_job run
+CMD bin/delayed_job run --queue=testable
```

Comment:
Revert this before merge - used to ensure testable workers pick up the right job